process-engine/bpmn-studio
# ✨ Make Diagrams Deployable via Solution Explorer

## Changes

1. Move Saving to Save Diagram Service
2. Move Deploying to Deploy Diagram Service
3. Add Context Menu Entry for Deploying
4. Fix Saving All Diagrams
5. Add 'Saving Diagrams' Modal

## Issues

Closes #1395
Closes #1710 

PR: #1696

## How to test the changes

- Check if saving/save as is still possible via 'Design'

- Check if deploying is still possible via 'Design'

- Right click on a diagram from a local solution
- Click on Deploy
- **Notice that the diagram will be deployed.**
